addons/mail/static/src/model/model_manager.js

```diff
@@ -1073,7 +1077,10 @@ class ModelManager {
             }
             const field = Model.__fieldMap[fieldName];
             if (!field) {
-                throw new Error(`Cannot create/update record with data unrelated to a field. (model: "${Model.modelName}", non-field attempted update: "${fieldName}")`);
+                throw new Error(`Cannot create/update record with data unrelated to a field. (model: "${record.localId}", non-field attempted update: "${fieldName}")`);
+            }
+            if (field.readonly && !allowWriteReadonly) {
+                throw new Error(`Can't update "${field.fieldName}" (model: "${record.localId}") because it's readonly.`);
```

Comment:
`model:` should be `record:` in these strings now that it prints the `localId` instead of the model name

Reply:
Still not done

Reply:
Still not done